### DISCUSSION TITLE
Bitbucket Branch and/or Pull Request building: which resource type implementation?

### DISCUSSION THREAD

**Discussion by ringods** [concourse/concourse#6361] [Help & Support]
Hello,

I found an question regarding Bitbucket branch and/or pull request building dating 2018 on the old discussion forum here:

https://discuss.concourse-ci.org/t/concourse-bitbucket-pull-request-support/94

None of the mentioned resources are listed on the new community resources page:

https://resource-types.concourse-ci.org/

So who is building branches and pull requests for their Bitbucket based git repositories and which resource type implementation are you using these days?

I need this to work with both Bitbucket Server and Bitbucket Cloud.

Thanks.

**Comment by taylorsilva** [comment]
> None of the mentioned resources are listed on the new community resources page:
>
>https://resource-types.concourse-ci.org/

We haven't gone out of our way to add resources to the site. We mainly wait for community contributions.

Back to your original question; I searched `concourse bitbucket resource` and found these that may be relevant for you:
- https://github.com/zarplata/concourse-git-bitbucket-pr-resource
- https://github.com/mmb/concourse-bitbucket-pullrequest-resource

Hope one of them works for you!
